kmr_mg-ud-test.conllu

```diff
@@ -6659,20 +6659,20 @@
 15	.	.	PUNCT	sent	_	3	punct	_	_
 
 # sent_id = wiki:wikibank.vislcg.txt:54:1468
-# text = Di 647an Iran, Lîbya hat dagir kirin û dagir kirina Anatoliya jî dest pê kir.
+# text = Di 647an Iran, Lîbya hat dagirkirin û dagir kirina Anatoliya jî dest pê kir.
```

Comment:
In passive voice "lêkerên hevdudanî" are written adjacent. [See](https://zanistvan.wordpress.com/wp-content/uploads/2020/12/pirtuken-zimane-kurdi-1.pdf) page 80. Page number written bottom of page.